src/main/java/com/blackducksoftware/integration/hub/notification/content/detail/NotificationContentDetailFactory.java

```diff
@@ -108,14 +112,16 @@ public NotificationContentDetailFactory(final Gson gson, final JsonParser jsonPa
                     componentValue = componentVersionStatus.component;
                 }
                 details.add(NotificationContentDetail.createDetail(content, contentType,
-                        createdAt, type, NotificationContentDetail.CONTENT_KEY_GROUP_POLICY, notificationState, content.projectName, content.projectVersionName, content.projectVersion, componentVersionStatus.componentName,
-                        componentValue, componentVersionStatus.componentVersionName, componentVersionStatus.componentVersion, policyName, policyUri, null, componentVersionStatus.componentIssueLink, null));
+                        createdAt, type, NotificationContentDetail.CONTENT_KEY_GROUP_POLICY, optionalNotificationState, Optional.of(content.projectName), Optional.of(content.projectVersionName), Optional.of(content.projectVersion),
```

Comment:
If any of these values can be null then we should use Optional.ofNullable() here

Reply:
The code has been fundamentally altered from this state, but your point is valid, even though it no longer applies here.